distribution/docs/src/main/resources/content/_appReferences/mg-geowebcache.adoc

```diff
@@ -137,7 +135,7 @@ Add a new Imagery Provider in the ${admin-console}:
 . Select *Standard Search UI* configuration.
 . Click the *Add* button next to *Imagery Providers*
 . Enter configuration for Imagery Provider in new textbox:
-. `{"type" "WMS" "url" "${secure_url}/geowebcache/service/wms" "layers" ["states"] "parameters" {"FORMAT" "image/png"} "alpha" 0.5}`
+. `{"type" "WMS" "url" "${secure_url}/geowebcache/service/wms" "layers" ["states"] "parameters" {"FORMAT" "image/png" "VERSION" "1.1.1"} "alpha" 0.5}`
```

Comment:
❗ There are also examples in https://github.com/codice/ddf/blob/2c280fca47752d4f0fe573827e815def461fd02d/distribution/docs/src/main/resources/content/_tables/search.standard.properties.adoc and https://github.com/codice/ddf/blob/12eb736abc9609426d88af7fa15a445ca0b44c77/distribution/docs/src/main/resources/content/_tables/ui.config%202.adoc that need to be update.